Rick&Morty/RickAndMorty/Scenes/Episodes/EpisodeList/EpisodeListView.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  EpisodeListView.swift
+//  RickAndMorty
+//
+//  Created by Oliver Jansta on 12.08.2022.
+//  Copyright © 2022 STRV. All rights reserved.
+//
+
+import SwiftUI
+
+struct EpisodeListView: View {
+    var body: some View {
+        ZStack {
+            BackgroundGradientView()
+            ScrollView() {
```

Comment:
You should specify the `Stack` that you want to use (even though it automatically creates one)